cloud/pkg/controller/manager/node.go

```diff
@@ -0,0 +1,38 @@
+package manager
+
+import (
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+)
+
+// NodesManager manage all events of nodes by SharedInformer
+type NodesManager struct {
+	events chan watch.Event
+}
+
+// Events return the channel save events from watch nodes change
+func (nm *NodesManager) Events() chan watch.Event {
+	return nm.events
+}
+
+// NewNodesManager create NodesManager by kube clientset and namespace
+func NewNodesManager(kubeClient *kubernetes.Clientset, namespace string) (*NodesManager, error) {
+	set := labels.Set{"node-role.kubernetes.io/edge": ""}
```

Comment:
its better if we define "node-role.kubernetes.io/edge" as a constant and use everywhere.